.github/workflows/web.yml

```
@@ -334,19 +334,19 @@ jobs:
       registry_password: ${{ secrets.DOCKER_REGISTRY_PASSWD }}
       composer_token: ${{ secrets.CENTREON_TECHNIQUE_PAT }}
 
-  legacy-e2e-test:
-    needs: [dockerize, changes]
-    if: ${{ !cancelled() && !contains(needs.*.result, 'failure') && !contains(needs.*.result, 'cancelled') && needs.changes.outputs.has_backend_changes == 'true' }}
-    uses: ./.github/workflows/behat-test.yml
-    with:
-      name: legacy-e2e
-      module_name: centreon
-      image_name: centreon-web
-      features_path: features
-    secrets:
-      registry_username: ${{ secrets.DOCKER_REGISTRY_ID }}
-      registry_password: ${{ secrets.DOCKER_REGISTRY_PASSWD }}
-      composer_token: ${{ secrets.CENTREON_TECHNIQUE_PAT }}
+  # legacy-e2e-test:
+  #   needs: [dockerize, changes]
+  #   if: ${{ !cancelled() && !contains(needs.*.result, 'failure') && !contains(needs.*.result, 'cancelled') && needs.changes.outputs.has_backend_changes == 'true' }}
+  #   uses: ./.github/workflows/behat-test.yml
+  #   with:
+  #     name: legacy-e2e
+  #     module_name: centreon
+  #     image_name: centreon-web
+  #     features_path: features
+  #   secrets:
+  #     registry_username: ${{ secrets.DOCKER_REGISTRY_ID }}
+  #     registry_password: ${{ secrets.DOCKER_REGISTRY_PASSWD }}
+  #     composer_token: ${{ secrets.CENTREON_TECHNIQUE_PAT }}
```

```suggestion
  legacy-e2e-test:
    needs: [dockerize, changes]
    if: ${{ !cancelled() && !contains(needs.*.result, 'failure') && !contains(needs.*.result, 'cancelled') && needs.changes.outputs.has_backend_changes == 'true' }}
    uses: ./.github/workflows/behat-test.yml
    with:
      name: legacy-e2e
      module_name: centreon
      image_name: centreon-web
      features_path: features
    secrets:
      registry_username: ${{ secrets.DOCKER_REGISTRY_ID }}
      registry_password: ${{ secrets.DOCKER_REGISTRY_PASSWD }}
      composer_token: ${{ secrets.CENTREON_TECHNIQUE_PAT }}
```
This should be uncommented